src/qtism/runtime/tests/RouteItem.php

```diff
@@ -426,4 +450,69 @@ public function getTimeLimits($excludeItem = false): RouteTimeLimitsCollection
 
         return $timeLimits;
     }
+
+    public function getEffectiveBranchRules(): BranchRuleCollection
+    {
+        if ($this->getBranchRules()->count() > 0) {
+            return $this->getBranchRules();
+        }
+
+        $sectionBranchRules = $this->getEffectiveSectionBranchRules();
+
+        if ($sectionBranchRules === null || $sectionBranchRules->count() > 0) {
```

Comment:
suggestion: `BranchRuleCollection` implements `Countable`.

```suggestion
        if ($sectionBranchRules === null || count($sectionBranchRules) > 0) {
```

Reply:
I want to avoid change it too much as is critical lib and I am getting underbudget for the ticket. So I will skip this changes for the moment if you do not mind :)